docs/installation/cloud-ex-aws.md

```diff
@@ -0,0 +1,196 @@
+
+      ![EC2 dashboard](images/ec2_launch_instance.png)
+
+      AWS EC2 virtual servers are called *instances* in Amazon parlance. Once you set up an account, IAM user and key pair, you are ready to launch an instance. It is at this point that you select the OS for the VM.
+
+  2. Choose an Amazon Machine Image (AMI) with the OS and applications you want. For this example, we select an Ubuntu server.
+
+      ![Launch Ubuntu](images/ec2-ubuntu.png)
+
+  3. Choose an instance type.
+
+      ![Choose a general purpose instance type](images/ec2_instance_type.png)
+
+  * Configure the instance.
+
+    You can select the default network and subnet, which are inherently linked to a region and availability zone.
+
+      ![Configure the instance](images/ec2_instance_details.png)
+
+  * Click **Review and Launch**.
+
+  * Select a key pair to use for this instance. (When you choose to launch, you need to select a key pair to use. Save the `.pem` file to use in the next steps. Save the )
+
+The instance is now up-and-running. The link path to get back to your EC2 instance on AWS is:
+
+    EC2 (Virtual Servers in Cloud) > EC2 Dashboard > Resources > Running instances
+
+Connect hints are available via the **Connect** button at the top of the AWS instance dashboard.
+
+
+### Step 3. Log in from a terminal, configure apt, and get packages
+
+1. Log in to the EC2 instance from a command line terminal.
+
+    Change directories into the directory containing the SSH key and run this command (or give the path to it as part of the command):
+
+        $ ssh -i "YourKey" ubuntu@xx.xxx.xxx.xxx
+
+    For our example:
+
+        $ cd ~/Desktop/keys/amazon_ec2
+      $ ssh -i "my-key-pair.pem" ubuntu@xx.xxx.xxx.xxx
+
+    We'll follow the instructions for installing Docker on Ubuntu at https://docs.docker.com/engine/installation/ubuntulinux/. The next few steps reflect those instructions.
+
+2. Check the kernel version to make sure it's 3.10 or higher.
+
+        ubuntu@ip-xxx-xx-x-xxx:~$ uname -r
+        3.13.0-48-generic
+
+3. Add the new gpg key
+
+        ubuntu@ip-xxx-xx-x-xxx:~$ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+        Executing: gpg --ignore-time-conflict --no-options --no-default-keyring --homedir /tmp/tmp.jNZLKNnKte --no-auto-check-trustdb --trust-model always --keyring /etc/apt/trusted.gpg --primary-keyring /etc/apt/trusted.gpg --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+        gpg: requesting key 2C52609D from hkp server p80.pool.sks-keyservers.net
+        gpg: key 2C52609D: public key "Docker Release Tool (releasedocker) <docker@docker.com>" imported
+        gpg: Total number processed: 1
+        gpg:               imported: 1  (RSA: 1)
+
+4. Create a `docker.list` file, and add an entry for our OS, Ubuntu Trusty 14.04 (LTS). (If we were updating an existing file, we'd delete any existing entries.)
```

Comment:
My preference is use follow on paras rather than parentheticals.  Provides space between what they have to do versus "information about what you are doing.  I'd be interested to learn what you think.